data/dataHelpers.js

```diff
@@ -0,0 +1,28 @@
+const knex = require("knex");
+const dbConfig = require("../knexfile.js");
+const db = knex(dbConfig.development);
+module.exports = {
+  addUser,
+  find,
+  findBy,
+  findById
+};
+
+function find() {
+  return db("users").select("id", "username", "password");
```

Comment:
You could simply opt for
```javascript
function find() {
  return db('users');
}
```
here since it defaults to selecting all of the columns of the given table (which your select method is doing, but with an extra step).